composer.json

```diff
@@ -93,27 +91,32 @@
         "prestashop/statssearch": "^2",
         "prestashop/statsstock": "^2",
         "prestashop/statsvisits": "^2",
-        "prestashop/translationtools-bundle": "^3",
+        "prestashop/translationtools-bundle": "dev-sf4-support",
         "prestashop/welcome": "^6",
-        "sensio/distribution-bundle": "^5.0.19",
-        "sensio/framework-extra-bundle": "^5.0.0",
+        "sensio/distribution-bundle": "^5.0",
+        "sensio/framework-extra-bundle": "^5.1",
         "shudrum/array-finder": "^1.1.0",
         "smarty/smarty": "^3.1.31",
         "soundasleep/html2text": "^0.5.0",
-        "swiftmailer/swiftmailer": "~5.4.3",
+        "swiftmailer/swiftmailer": "~6.2",
         "symfony/monolog-bundle": "^3.1.0",
         "symfony/polyfill-apcu": "^1.0",
         "symfony/polyfill-php73": "^1.10",
-        "symfony/swiftmailer-bundle": "^2.6.4",
-        "symfony/symfony": ">=3.4.8",
+        "symfony/swiftmailer-bundle": "^3.1",
+        "symfony/symfony": "^3.4",
+        "symfony/twig-bridge": "^3.4",
         "tecnickcom/tcpdf": "^6.2.12",
         "tijsverkoyen/css-to-inline-styles": "^2.2",
-        "twig/twig": "^1.38||^2.6.2"
+        "twig/twig": "^2.6.2"
     },
     "repositories": [
         {
             "type": "vcs",
             "url": "https://github.com/prestashop/php-cssjanus"
+        },
+        {
+            "type": "vcs",
+            "url": "https://github.com/mickaelandrieu/TranslationToolsBundle"
```

Comment:
waiting for https://github.com/PrestaShop/TranslationToolsBundle/pull/57 to be merged

Reply:
solved! thanks @eternoendless